File: app/src/main/java/com/cheatdatabase/listitems/GameListItem.java
package com.cheatdatabase.listitems;

import com.cheatdatabase.data.model.Game;

public class GameListItem extends ListItem {

    private Game game;

    public Game getGame() {
        return game;
    }

    public void setGame(Game game) {
        this.game = game;
    }

    @Override
    public int getType() {
        return ListItem.TYPE_GAME;
    }

    @Override
    public String getTitle() {
        return game.getGameName().substring(0, 1);
    }
}

File: app/src/main/java/com/cheatdatabase/listitems/GameListItem.kt
package com.cheatdatabase.listitems

import com.cheatdatabase.data.model.Game

class GameListItem : ListItem() {
    var game: Game? = null

    override fun type(): Int {
        return TYPE_GAME
    }

    override fun title(): String? {
        return game?.gameName?.substring(0, 1)
    }
}